test-suite/output/bug_15097.out

```diff
@@ -0,0 +1,2 @@
+The command has indeed failed with message:
+Cannot find a physical path bound to logical path matching suffix Coq.
```

Comment:
This error is very strange tbh, the problem is DoesNotExist but all it mentions is Coq.
Why don't we just say `cannot find Coq.DoesNotExist` (`cannot find DoesNotExist with prefix Coq` if using From)?

Reply:
I've reworded the message and changed the information now.